admin/src/components/Login/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Login extends React.Component {
+  state = {
+    isLoading: false
+  };
+
+  onLogin(e, auth) {
+    e.preventDefault();
+    this.setState({ isLoading: true });
+    auth(this.refs.email.value, this.refs.password.value);
+  }
+
+  render() {
+    const { auth } = this.props;
```

Comment:
this should be named onLoginSubmit or handleLoginSubmit, I want to enforce a bit of standardization of function naming through our react code at least

Reply:
`onLoginSubmit`